apps/labs/posts/czi-eoss-5-conda-forge.md

```diff
@@ -0,0 +1,286 @@
+
+> **3. Development of a maintenance dashboard on Quetz**
+
+> There is no straightforward way to monitor the operational status of conda-forge or Bioconda's infrastructure. The existing [conda-forge.org/status panel](https://conda-forge.org/status/) is far from giving a comprehensive view of ongoing maintenance tasks, bottlenecks or the overall health of the many bots and infrastructure pieces. Having a detailed picture of the infrastructure and automation tools will significantly improve the maintainers' workflow and aid with identifying critical risks. The dashboard will be built on top of [Quetz](https://quetz.readthedocs.io/en/latest/index.html), an open source server for hosting conda packages, allowing for increased transparency and extensibility.
+
+Over these two years, the Quansight and QuantStack teams involved the following people:
+
+- Quansight: Tania Allard, Jaime Rodríguez-Guerra, Vinicius D. Cerutti, Gonzalo Peña-Castellanos, Isabella Presedo-Floyd, Axel Obermeier, Isuru Fernando, Sophia Castellarin.
+- QuantStack: Sylvain Corlay, Gabriela Vives, Hind Montassif, Johan Mabille, Afshin Darian, Florence Haudin, Vanessa Sochat.
+
+The proposal was scoped so that QuantStack team would lead the OCI mirroring efforts, while the Quansight team would tackle the infrastructure details. The maintenance dashboard and website improvements would be split roughly 50-50 between the two teams.
+
+As we started working on this project, we realized there were many synergies from other funding sources that could be aligned together for maximum output. So in the next few sections you will also learn about other initiatives that ended up scoped together, like the new website or the GPU CI server.
+
+## Website and maintenance dashboard revamp
+
+The [conda-forge.org website](https://conda-forge.org) used to be built around its Sphinx documentation, with several add-ons coming in from different repositories that ended up being "mounted" on the main conda-forge.org deployment.
+While functional, this approach made the site difficult to maintain and led to many inconsistencies in design, usability, and contribution workflows.
+
+<div style={{ display: 'flex', justifyContent: 'center', gap: '20px', alignItems: 'flex-start' }}>
+  <figure style={{ textAlign: 'center', flex: 1, margin: 0 }}>
+    <a href="/posts/czi-eoss-5-conda-forge/conda-forge-old-1.png" target="_blank">
+    <img 
+      src="/posts/czi-eoss-5-conda-forge/conda-forge-old-1.png"
+      alt="Screenshot of the frontpage of the original conda-forge website. A forge emitting impact sparks is used as a dark background. A prominent large white text reading conda-forge sits below the official conda-forge logo, a black anvil. Some smaller text follows, also in white."
```

Comment:
@trallard I assume I also need to rework these alt captions by e.g. removing "Screenshot of" and similar?

Reply:
yes please